miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/errors/MiniAppError.kt

```diff
@@ -0,0 +1,16 @@
+package com.rakuten.tech.mobile.miniapp.errors
+
+import androidx.annotation.Keep
+
+/**
+ *  Contains the components to use custom errors from host app.
+ *  @property type The type of error.
+ *  @property message error message send to min app.
+ */
+@Keep
+data class MiniAppError(
+    val type: String = "",
+    val message: String = ""
+) {
+    constructor() : this(type = "", message = "")
```

Comment:
I think this constructor isn't needed for anything, right?

Reply:
yes, i will remove this constructor.. @corycaywood